src/utilities/__tests__/findBreakingChanges-test.js

```diff
@@ -574,215 +451,81 @@ describe('findBreakingChanges', () => {
-            },
-            arg6: {
-              type: GraphQLNonNull(GraphQLString),
-            },
-            arg7: {
-              type: GraphQLNonNull(GraphQLList(GraphQLInt)),
-            },
-            arg8: {
-              type: GraphQLInt,
-            },
-            arg9: {
-              type: GraphQLList(GraphQLInt),
-            },
-            arg10: {
-              type: GraphQLList(GraphQLNonNull(GraphQLInt)),
-            },
-            arg11: {
-              type: GraphQLList(GraphQLInt),
-            },
-            arg12: {
-              type: GraphQLList(GraphQLList(GraphQLInt)),
-            },
-            arg13: {
-              type: GraphQLNonNull(GraphQLInt),
-            },
-            arg14: {
-              type: GraphQLList(GraphQLNonNull(GraphQLList(GraphQLInt))),
-            },
-            arg15: {
-              type: GraphQLList(GraphQLNonNull(GraphQLList(GraphQLInt))),
-            },
-          },
-        },
-      },
-    });
-
-    const newType = new GraphQLObjectType({
-      name: 'Type1',
-      fields: {
-        field1: {
-          type: GraphQLString,
-          args: {
-            arg1: {
-              type: GraphQLInt,
-            },
-            arg2: {
-              type: GraphQLList(GraphQLString),
-            },
-            arg3: {
-              type: GraphQLString,
-            },
-            arg4: {
-              type: GraphQLNonNull(GraphQLString),
-            },
-            arg5: {
-              type: GraphQLInt,
-            },
-            arg6: {
-              type: GraphQLNonNull(GraphQLInt),
-            },
-            arg7: {
-              type: GraphQLList(GraphQLInt),
-            },
-            arg8: {
-              type: GraphQLNonNull(GraphQLList(GraphQLInt)),
-            },
-            arg9: {
-              type: GraphQLList(GraphQLNonNull(GraphQLInt)),
-            },
-            arg10: {
-              type: GraphQLList(GraphQLInt),
-            },
-            arg11: {
-              type: GraphQLList(GraphQLList(GraphQLInt)),
-            },
-            arg12: {
-              type: GraphQLList(GraphQLInt),
-            },
-            arg13: {
-              type: GraphQLNonNull(GraphQLList(GraphQLInt)),
-            },
-            arg14: {
-              type: GraphQLList(GraphQLList(GraphQLInt)),
-            },
-            arg15: {
-              type: GraphQLList(
-                GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLInt))),
-              ),
-            },
-          },
-        },
-      },
-    });
-
-    const oldSchema = new GraphQLSchema({
-      query: queryType,
-      types: [oldType],
-    });
-
-    const newSchema = new GraphQLSchema({
-      query: queryType,
-      types: [newType],
-    });
+    const oldSchema = buildSchema(`
+      type Type1 {
+        field1(
+          arg1: String, arg2: String, arg3: [String], arg4: String, arg5: String!, arg6: String!, arg7: [Int]!,
```

Comment:
nit: one argument per line without commas would be consistent with prettier:
https://github.com/prettier/prettier/blob/master/tests/graphql_arguments/__snapshots__/jsfmt.spec.js.snap#L12-L16